### ISSUE TITLE
Avoid long trace file names 

### ISSUE THREAD

**Issue by bstabler** [ActivitySim/activitysim#409]
| Idea          | Level-of-effort | Notes |  Priority | 
| ------------- | ------------- |   ------------- | ---- |
| Avoid long trace file names | Hours | Reduce filenames (especially in estimation mode and for tracing) because of Windows filename limits. | Low |

**Comment by bstabler** [comment]
@stefancoe - it would be good to create folders too to help organize all the output files

**Comment by bstabler** [comment]
It's pretty simple to turn on support for long file names on Windows 10.  See https://www.howtogeek.com/266621/how-to-make-windows-10-accept-file-paths-over-260-characters/ and:

![image](https://user-images.githubusercontent.com/5761221/113368389-7b772680-9313-11eb-88f9-13e869973eef.png)

I did this, but I also needed to turn off the assertation in activitysim so we should make this assertation optional and user configurable.

**Comment by bettinardi** [comment]
Not all government employees have "easy" access to their registry files or other admin level changes.  Additionally, this is going to be one of those really frustrating things to remember anytime someone gets a new machine.  I recommended attempting to find a solution that works in Windows default.

**Comment by bstabler** [comment]
Agreed.  Another work around is to setup the model in a short folder name location such as c:\activitysim.

**Comment by bettinardi** [comment]
I don't think this issue should be resolved with work arounds.  The issue should be resolved with a fix/soultion.

**Comment by guyrousseau** [comment]
As for file name length, ARC is a little more hesitant with this one.  The length requirement was originally set because of the constraints of binary itself (255) and consistent across many operating systems.  While modern operating systems are capable of addressing outside this original bound, not all client software installed and running on these operating systems are capable of addressing files at extended length.   ARC would proceed with great caution, as there is potential for data loss or corruption, should the aforementioned software packages on the servers and management systems, such as ARC's backup software, prove unable to address files at the length specified above.
